utils/dragster.test.ts

```diff
@@ -52,36 +52,56 @@ describe('utils.dragster', () => {
     it('should dispatch dragenter event', () => {
         div.dispatchEvent(dragenter);
 
-        expect(enterEvent.detail.detail).toEqual('dragenter_detail');
+        if (enterEvent === null) {
```

Comment:
Rather than doing this explicit null check, you can use the `!.` operator to do an implicit null check.
See this for more info: https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

Reply:
Ah that's a sweet feature that I wasn't aware of. Tried it out though and unfortunately looks like `typescript-eslint` isn't a fan of the feature: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-non-null-assertion.md

Any thoughts on modifying the eslint rules?

Reply:
@devinbinnie Pinging for feedback

Reply:
Rebased with eslint config changes and fixed.